internal/postgres/config_test.go

```diff
@@ -175,7 +175,8 @@ func TestBashRecreateDirectory(t *testing.T) {
 	cmd.Args = append(cmd.Args, "-ceu", "--",
 		bashRecreateDirectory+` recreate "$@"`, "-",
 		filepath.Join(dir, "d"), "0740")
-
+	// Set LANG=C to ensure that CombinedOutput is correctly ordered
+	cmd.Env = append(cmd.Env, "LANG=C")
```

Comment:
🤔 I'm confident the test failed because an uninitialized `cmd.Env` is replaced with a copy of the environment of `go test`. The implementation doesn't care about the environment, only the test. We want the test to behave consistently regardless of the environment.

🤔 I see that `cmd.Env = []string{}` also causes the test to pass for me. I _think_ this is a more direct solution ("ignore the environment while testing") but I'm also finding it harder to describe in a comment. Maybe the following?

```go
	// The assertion below expects alphabetically sorted filenames.
	// Set an empty environment to always use the default/standard locale.
	cmd.Env = []string{}
```